README.md

```diff
@@ -4,20 +4,41 @@
 
 ## Detecting system-wide changes in whole transcriptomes
 
-We have previously described the ability of a machine learning classifier to detect an
-NF1 inactivation signature using Glioblastoma data. We applied an ensemble of logistic
-regression classifiers to the problem, but the solutions were unstable and overfit. To
-address these issues, we posited that we could leverage data from diverse tissue-types
-to build a pancancer NF1 classifier. We also predicted that an NF1/RAS classifier would
-better key in on NF1 inactivation since NF1 directly inhibits RAS activity and there
-are many more examples of samples with RAS mutations.
+We have previously described the ability of a machine learning classifier to
+detect an NF1 inactivation signature using Glioblastoma data
+([Way _et al._ 2016](http://dx.doi.org/10.1101/075382)). We applied an ensemble
+of logistic regression classifiers to the problem, but the solutions were
+unstable and overfit. To address these issues, we posited that we could leverage
+data from diverse tissue-types to build a pancancer NF1 classifier. We also
+predicted that an NF1/RAS classifier would better key in on NF1 inactivation
+since NF1 directly inhibits RAS activity and there are many more examples of
+samples with RAS mutations.
+
+The code in this repository is flexible and can build a Pan-Cancer classifier
+for any combination of genes and cancer types using gene expression, mutation,
+and copy number data.
 
 ## Controlled Access Data
 
-All data used in this analysis are under controlled access by the The National Institutes
-of Health (NIH) and The Cancer Genome Atlas (TCGA). All data are downloaded from
-[synapse](http://synapse.org), which requires login and access credentials. To request
-access contact _SynapseInfo@sagebase.org_ for specific details and instructions.
+All data used in this analysis are under controlled access by the The National
+Institutes of Health (NIH) and The Cancer Genome Atlas (TCGA). All data are
+downloaded from [synapse](http://synapse.org), which requires login and access
+credentials. To request access contact _SynapseInfo@sagebase.org_ for specific
+details and instructions.
+
+## Publicly Available Data
+
+We also provide the ability to build classifiers using publicly available
+data retrieved from [UCSC Xena](xena.ucsc.ecu).
+
+## Cancer Genes
+
+Note that in order to use the copy number integration feature, an additional
+file must be downloaded. The file is `Supplementary Table S2` of
+[Vogelstein _et al._ 2013]("http://doi.org/10.1126/science.1235122"). It can
+be downloaded [here]("science.sciencemag.org/highwire/filestream/594203/field_highwire_adjunct_files/1/1235122TablesS1-4.xlsx").
```

Comment:
Wouldn't it be better to script this? Even a one line shell command you put in the README here.